### DISCUSSION TITLE
PWA MODE doesn't installable

### DISCUSSION THREAD

**Discussion by 1377sayaba** [quasarframework/quasar#15274] [CLI - PWA mode] [answered]
### PWA MODE doesn't installable 
hello 
at the start I want say thanks for amazing quasar .
[report of lighthouse ](https://s27.picofile.com/file/8458468776/download.pdf.html)
I have a problem in my projects that PWA MODE doesn't installable 
can i reinstall PWA mode ?

**Comment by suleimanas** [comment]
hello, you need to provide more information
do you use webpack or vite
is PWA mode installed - do you have src-pwa folder in your project
do you have your pwa configs and manifest already setup in quasar.config

**Comment by 1377sayaba** [comment]
![image](https://user-images.githubusercontent.com/89265632/212492716-4f15355e-2e25-47a2-9b43-ab928d0f9751.png)
![image](https://user-images.githubusercontent.com/89265632/212492783-845d945c-e144-4b51-970a-5eed4f8d9697.png)
i use webpack
is PWA mode installed - do you have src-pwa folder in your project ? yes
do you have your pwa configs and manifest already setup in quasar.config ? yes

**Reply by rstoenescu** [reply]
Post a repo and the output of "quasar info" command inside of your project folder.
Or:

![image](https://user-images.githubusercontent.com/4665641/213199042-c4809fda-194c-460f-a7e8-a87546d132b7.png)

**Reply by 1377sayaba** [reply]
thanks for your response 
For the sake of code protection, I have emailed items with the same title as this thread

**Comment by rstoenescu** [comment, marked as answer]
It seems that you have manually added `/public/icons/site.webmanifest` and linked to it in `/src/index.template.html`. This interferes with the Quasar generated service worker (through workbox). Remove the file and `<link rel="manifest" href="icons/site.webmanifest" />` from the html file.

**Reply by 1377sayaba** [reply]
I removed the interference
But only the errors changed
![download2 (1)_1](https://user-images.githubusercontent.com/89265632/213678884-8fe6566f-0383-4ad1-bbbe-6e06e0fd9078.jpg)
![download2 (1)_2](https://user-images.githubusercontent.com/89265632/213678931-c2bc639a-fda2-4269-8c01-2458cd2f461a.jpg)
my manifest:
`{
  "name": "Suppliers of Bamachoub",
  "short_name": "بامافروش",
  "description": "وب سایت تامین کنندگان فروشگاه باماچوب",
  "display": "standalone",
  "start_url": "https://supplier.bamachoub.com",
  "icons": [
    {
      "src": "/icons/icon-128x128.png",
      "sizes": "128x128",
      "type": "image/png",
      "purpose": "maskable"
    },
    {
      "src": "/icons/ms-icon-144x144.png",
      "sizes": "144x144",
      "type": "image/png",
      "purpose": "maskable"
    },
    {
      "src": "/icons/icon-192x192.png",
      "sizes": "192x192",
      "type": "image/png",
      "purpose": "maskable"
    },
    {
      "src": "/icons/icon-256x256.png",
      "sizes": "256x256",
      "type": "image/png",
      "purpose": "maskable"
    },
    {
      "src": "/icons/icon-384x384.png",
      "sizes": "384x384",
      "type": "image/png",
      "purpose": "maskable"
    },
    {
      "src": "/icons/icon-512x512.png",
      "sizes": "512x512",
      "type": "image/png",
      "purpose": "maskable"
    }
  ],
  "orientation": "portrait",
  "background_color": "#ffffff",
  "theme_color": "#50459c",
  "prefer_related_applications": false
}
`
[my site](https://supplier.bamachoub.com/)
